native/acter/src/api/profile.rs

```diff
@@ -111,6 +112,14 @@ impl RoomProfile {
         RoomProfile { room }
     }
 
+    pub fn room_id(&self) -> OwnedRoomId {
```

Comment:
Here it goes again...I'm not sure if ```RoomId``` provides anything useful other than being a identifier which ```room_id_str``` does it.